validation.sh

```diff
@@ -11,4 +11,8 @@
 set -e
 
 DIR="$(dirname "$0")"
-"$DIR/run.sh" "$DIR/src/run_validation.py" $@
+if [  -n "$(uname -a | grep Ubuntu)" ]; then
+    "sudo" "$DIR/run.sh" "$DIR/src/run_validation.py" $@
```

Comment:
I don't think you need to do this. @peterzhuamazon  Can add more but entire code base runs as it is irrespective of sudo. Example: https://github.com/opensearch-project/opensearch-build/blob/main/src/test_workflow/integ_test/distribution_rpm.py#L29-L32

Reply:
Added my comments.